frontend/src/actions/authActions.ts

```diff
@@ -35,10 +36,12 @@ export const signUserUp = (payload: IUserDetails) => async (dispatch:Dispatch<fe
 
   try {
     const result = await authApiClient.register(payload)
-    localStorage.setItem('token', result.token)
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    //@ts-ignore
+    localStorage.setItem('token', result.token.access)
     dispatch(setUserAction(result.user))
   } catch (e) {
-    console.log('Error:', e)
+    alert(e.message)
```

Comment:
Alert's aren't good approach, but for now let's leave it. You can create a ticket to develop a better user-friendly notification system.
For now, let's do the next this: alert should just say "Sorry, an error occurred, try again later". And console.log(e.message).
This will be more correct for UX - users should not care about error content if they can't fix it.